rebar.config

```diff
@@ -26,7 +26,7 @@
 
 {deps, [
   {sumo_db, "0.6.4"},
-  {riakc, "2.5.3-beta1"},
+  {riakc, {git, "https://github.com/basho/riak-erlang-client.git", {tag, "2.5.3"}}},
```

Comment:
Hi @frepond, I understand, that's annoying, but that's an Inaka's rule, as long as the hex version exist, we should work with that :)